analyses/immune-deconv/run-immune-deconv.sh

```diff
@@ -1,54 +1,47 @@
 #!/bin/bash
-# Module author: Komal S. Rathi
-# 2022
+# Module author: Komal S. Rathi, updated Kelsey Keith
+# 2022-07
 
-# This script runs the steps for immune deconvolution using xCell and CIBERSORT (absolute mode). 
+# This script runs the steps for immune deconvolution using xCell and quanTIseq. 
 
 set -e
 set -o pipefail
 
-# Run original files - will not by default
-RUN_CIBERSORT=${RUN_CIBERSORT:-0}
-
 # This script should always run as if it were being called from
 # the directory it lives in.
 script_directory="$(perl -e 'use File::Basename;
   use Cwd "abs_path";
   print dirname(abs_path(@ARGV[0]));' -- "$0")"
 cd "$script_directory" || exit
 
-# xCell
+### xCell
 # generate deconvolution output
-echo "Deconvolution..."
+echo "Deconvolution xCell..."
 Rscript --vanilla 01-immune-deconv.R \
---expr_mat '../../data/gene-expression-rsem-tpm-collapsed.rds' \
---clin_file '../../data/histologies.tsv' \
+--expr_mat '../../data/v11/gene-expression-rsem-tpm-collapsed.rds' \
+--clin_file '../../data/v11/histologies.tsv' \
```

Comment:
please set these to use the data folder directly - I think this is why the module is failing